make-webpack-config.js

```diff
@@ -169,7 +172,7 @@ module.exports = function(options) {
         filename: baseFilename + fileExtension,
         chunkFilename: buildType === 'dev' ? '[id].js' : '[chunkhash].js',
         sourceMapFilename: '[file].map',
-        library: buildType === 'host' ? 'Smooch' : undefined,
+        library: undefined,
```

Comment:
Webpack doesn't need to expose a global variable anymore, it's handled by the script loader.